packages/web-config-server/src/apiV1/index.js

```diff
@@ -39,30 +30,21 @@ const handleWith = Handler =>
 export const getRoutesForApiV1 = () => {
   const api = Router();
   // mount the routes
-  api.get('/getUser', catchAsyncErrors(getUser()));
-  api.post('/login', catchAsyncErrors(login));
-  api.post('/login/oneTimeLogin', catchAsyncErrors(oneTimeLogin));
-  api.get('/logout', catchAsyncErrors(logout));
-  api.post('/signup', catchAsyncErrors(appSignup()));
-  api.post('/changePassword', catchAsyncErrors(appChangePassword()));
-  api.post('/resetPassword', catchAsyncErrors(appRequestResetPassword()));
-  api.get('/downloadFiles', catchAsyncErrors(appDownloadFiles()));
```

Comment:
Looks like this one was only added in June last year, but as far as I can tell the only usage of it is in `tupaia-web`, where `tupaia-web-server` handles it (by forwarding it to `central-server`)

Reply:
I think `tupaia-web-server` might be forwarding quite a few things to `web-config-server` by using the `*` route, which I have a suspicion would include these auth routes (although looks like not the `downloadFiles` route). Would be worth explicitly forwarding anything that needs to be forwarded, and using central-server where possible if there is already logic in there for a route

Reply:
Done for `tupaia-web-server`. Looks like `datatrak-web-server` doesn’t forward anything to `web-config-server` like this. There are just a couple instances where it defines its own handler that uses `ctx.services.webConfig`